streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/TableProcessorNode.java

```diff
@@ -17,36 +17,32 @@
 
 package org.apache.kafka.streams.kstream.internals.graph;
 
-import org.apache.kafka.common.utils.Bytes;
-import org.apache.kafka.streams.kstream.internals.KeyValueStoreMaterializer;
-import org.apache.kafka.streams.kstream.internals.MaterializedInternal;
-import org.apache.kafka.streams.processor.StateStore;
 import org.apache.kafka.streams.processor.internals.InternalTopologyBuilder;
 import org.apache.kafka.streams.state.KeyValueStore;
+import org.apache.kafka.streams.state.StoreBuilder;
 
 import java.util.Arrays;
 
-public class TableProcessorNode<K, V, S extends StateStore> extends StreamsGraphNode {
+public class TableProcessorNode<K, V> extends StreamsGraphNode {
 
-    private final MaterializedInternal<K, V, S> materializedInternal;
     private final ProcessorParameters<K, V> processorParameters;
     private final String[] storeNames;
+    private final StoreBuilder<KeyValueStore<K, V>> storeBuilder;
 
     public TableProcessorNode(final String nodeName,
```

Comment:
Although the constructor was pre-existing, I'm thinking we could clean things up a little bit by adding a constructor
```java
 public TableProcessorNode(final String nodeName,
        final ProcessorParameters<K, V> processorParameters,
        final StoreBuilder<KeyValueStore<K, V>> storeBuilder) {
        
        this(nodeName, processorParameters, null, storeBuilder);
    }
```
Then it's more clear in the code when we call 
```java
final StreamsGraphNode tableNode = new TableProcessorNode<>(
            name,
            processorParameters,
            storeBuilder
        );
```
And we can leave the existing constructor using all 4 parameters alone.